service/pom.xml

```diff
@@ -24,6 +24,13 @@
       <artifactId>dropwizard-core</artifactId>
       <version>${dropwizard.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>net.sf.json-lib</groupId>
+      <artifactId>json-lib</artifactId>
+      <version>2.4</version>
+      <classifier>jdk15</classifier>
+    </dependency>
```

Comment:
I'm interested to hear more, because I had this problem in Quip, and had to solve it by adding the dependency.  When I run mvn clean package I'm only trying to build the service, is there a more specific mvn command for that?